modal/functions.py

```diff
@@ -196,7 +196,7 @@ async def _stream_function_call_data(
     while True:
         req = api_pb2.FunctionCallGetDataRequest(function_call_id=function_call_id, last_index=last_index)
         try:
-            async for chunk in unary_stream(stub_fn, req):
+            async for chunk in unary_stream(stub_fn, req):  # type: ignore[var-annotated]
```

Comment:
@freider I'm a little puzzled why this now raises a mypy error, since `unary_stream` returns `AsyncIterator[_RecvType]` https://github.com/modal-labs/modal-client/blob/83e90f273bfcb6b10c568f17bf7baca0e366e903/modal_utils/grpc_utils.py#L205.

Not sure why this isn't picked up in `functions.py`.

Reply:
I think it's due to `--follow-imports=skip`, so the imported type isn't available (removing that flag, this error disappears but you get tons of errors *in the imported files* instead...). I wonder if there is a way to check a single file for type errors but still follow imports in order to get types